internal_ws/ykcompile/src/stack_builder.rs

```diff
@@ -1,3 +1,10 @@
+//! A stack builder for allocating objects on the stack.
+//!
+//! The builder is naive in that it allocates objects immediately (when `alloc` is called), thus
+//! potentially inserting more alignment padding than is necessary.
```

Comment:
s/than is necessary/than would be necessary if we deferred and reordered allocations/ ?